packages/superset-ui-chart/src/components/SuperChart.tsx

```diff
@@ -37,6 +62,27 @@ export default class SuperChart extends React.PureComponent<Props, {}> {
 
   private createChartProps = ChartProps.createSelector();
 
+  private parseDimension = createSelector(
+    ({ width }: { width: string | number; height: string | number }) => width,
+    ({ height }) => height,
+    (width, height) => {
+      // Parse them in case they are % or 'auto'
+      const widthInfo = parseLength(width);
+      const heightInfo = parseLength(height);
+
+      const style = {
+        height: heightInfo.isDynamic ? '100%' : heightInfo.value,
```

Comment:
could this be simplified so `parseLength` sets `value` to `100%` if `isDynamic` is true?

Reply:
if not, below we use `multiplier`, why not use `multiplier` here instead of `100%` or is this slightly different?

Reply:
Great idea.